docs/get_started.md

```diff
@@ -28,51 +28,75 @@
- 
-   <div id="GSG_nextstepchoice">
-     <a href="tutorials.html" >
-        <h4>Interactive Tutorials - Jupyter Notebooks (Python) </h4>
-        <p>Learn from a choice of interactive Python tutorials targeting typical OpenVINO use cases.</p>
-     </a> 		
-     <a href="openvino_docs_OV_UG_Samples_Overview.html" >
-        <h4>OpenVINO Samples	</h4>
-        <p>See ready-made applications explaining OpenVINO features and various use-cases.		</p>
-     </a> 
-     <a href="openvino_inference_engine_ie_bridges_python_sample_speech_sample_README.html" >
-        <h4>Reference Implementation For Speech Recognition Apps (Python)</h4>
-        <p>Use a speech recognition demo and Kaldi model conversion tool as reference. </p>
-     </a>
-     <a href="openvino_inference_engine_samples_speech_sample_README.html" >
-        <h4>Reference Implementation For Speech Recognition Apps (C++)</h4>
-        <p>Use a speech recognition demo and Kaldi model conversion tool as reference. </p>
-     </a>
-     <a href="omz_demos.html" >
-        <h4>Open Model Zoo Demo Applications</h4>
-        <p>See Open Model Zoo demo applications to use robust application templates for implementing specific deep learning scenarios. </p>
-     </a>
-     <a href="http://devcloud.intel.com/edge/" >
-        <h4>Intel® DevCloud 	</h4>
-        <p>Develop, test, and run your OpenVINO solution for free on a cluster of the latest Intel® hardware. </p>
-     </a> 
-   </div>
-   <div style="clear:both;"> </div>
-
-@endsphinxdirective
\ No newline at end of file
+Visit the <a href="openvino_docs_install_guides_overview.html">Install OpenVINO Overview page</a> to view options for installing OpenVINO and setting up a development environment on your device.
+   
+## <a name="get-started-tutorials"></a>3. Learn OpenVINO
+   
+OpenVINO provides a wide array of examples and documentation showing how to work with models, run inference, and deploy applications. Step through the sections below to learn the basics of OpenVINO and explore its advanced optimization features. Visit <a href="documentation.html">OpenVINO’s documentation</a> for further details on how to use its features and tools.
+   
+OpenVINO users of all experience levels can try <a href="https://www.intel.com/content/www/us/en/developer/tools/devcloud/edge/overview.html">Intel Dev Cloud</a>, a free web-based service for developing, testing, and running OpenVINO applications for free on an online cluster of the latest Intel hardware.
+
+
+### <a name="openvino-basics"></a>OpenVINO Basics
+Learn the basics of working with models and inference in OpenVINO. Begin with “Hello World” Interactive Tutorials that show how to prepare models, run inference, and retrieve results using the OpenVINO API. Then, explore other examples from the Open Model Zoo and OpenVINO Code Samples that can be adapted for your own application.
+   
+
+#### <a href="tutorials.html"><ins>Interactive Tutorials - Jupyter Notebooks</ins></a>
```

Comment:
I want each header underneath the OpenVINO Basics section to a link. Here's the page each header should link to:

- "Interactive Tutorials - Jupyter Notebooks"   should link to the [Tutorials home page](https://docs.openvino.ai/latest/tutorials.html)
- "OpenVINO Code Samples"  should link to the [Samples page](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Samples_Overview.html)
- "Integrate OpenVINO With Your Application"   should link to the [Integrate OpenVINO With Your Application](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Integrate_OV_with_your_application.html) page

I would like each link to be bold and underlined as shown in the image below, because it fits well with the style of the rest of the page. 

@kblaszczak-intel how should I implement the links to accomplish this?

![image](https://user-images.githubusercontent.com/26154245/203152475-1f605321-e231-4143-ae95-74d92c81f4bf.png)

Reply:
I went ahead and removed the links from the headings and moved them into the sentences below each heading.